onset_maproom/calc.py

```diff
@@ -4,6 +4,22 @@
 
 # Date Reading functions
 def read_zarr_data(zarr_path):
+    """ Read and return data in zarr format.
```

Comment:
consistently 3rd person or infinitive. I'd say 3rd person rather than infinitive but it's probably not my place to comment too much on English. Consistency would be good though, I think.

Reply:
[PEP257](https://peps.python.org/pep-0257/#one-line-docstrings) says:

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.